### ISSUE TITLE
Parameter of CM model setting issues for plug-in tasks in the real world

### ISSUE THREAD

**Issue by yolo01826** [Aaditya-Prasad/consistency-policy#3]
Thank you very much for your work. Previously, under your guidance, we successfully completed the plug-in task using the original DP. Following the same parameter configuration from your consistency policy in the simulation task, we trained the EDM and CTMP strategies for the plug-in task. However, the resulting CTMP curve appears quite strange. Could you advise if there are any changes needed in the parameter settings for strategies in the real environment?
EDM training curve as shown in the attached image.
![image](https://github.com/user-attachments/assets/e53294b2-c824-4abc-b41a-f6afc7cf4fe8)

CTMP training curve as shown in the attached image.
![image](https://github.com/user-attachments/assets/204e44e7-38e3-4cce-925f-ba197ba87920)
![image](https://github.com/user-attachments/assets/ee98751e-72f7-4101-ac75-7c36c12b5acd)
![image](https://github.com/user-attachments/assets/1d18b3f5-9224-4675-b092-a692e5a0620a)

Thank you!

**Comment by yolo01826** [comment]
@kevin-thankyou-lin @Aaditya-Prasad

**Comment by Aaditya-Prasad** [comment]
I have had runs with loss 'curves' that look like what you're seeing; I copied loss and mse error from one of our real world training runs below:
![image](https://github.com/user-attachments/assets/4b2ec298-8aed-4590-a902-ae5263ab2a5c)

We have had runs (in sim) without the loss spikes results (though you can see overfitting): 
![image](https://github.com/user-attachments/assets/c4fd853b-b977-481e-bdba-ef85da0acb04)

The loss spikes do appear only in the DSM loss (1), the CTM loss (2) is much more regular: 
![image](https://github.com/user-attachments/assets/2e39fe1b-88e2-48e4-817c-5f3c5c1f7e13)

![image](https://github.com/user-attachments/assets/cb5437d9-df4d-4612-9c02-1bcde14b2aca)

so the first thing I'd try would be to log the sampled noise scales for the DSM loss and see if the spikes happen when very ood noise scales are sampled. I didn't get a chance to try this due to timelines. It'd also be reasonable to play around with the learning rate, relative loss balancing, and making sure the dataset is properly shuffled.
